Still failing on python310.

```
datadog> FAILED tests/integration/dogshell/test_dogshell.py::TestDogshell::test_comment - AssertionError: b''
datadog> FAILED tests/integration/dogshell/test_dogshell.py::TestDogshell::test_event - Exception: Retry limit reached for command ['event', 'show', '5309722366481...
datadog> FAILED tests/integration/dogshell/test_dogshell.py::TestDogshell::test_metrics - Exception: Retry limit reached for command ['search', 'query', '088c10ad5aa...
datadog> FAILED tests/integration/dogshell/test_dogshell.py::TestDogshell::test_timeboards - AssertionError: b''
datadog> FAILED tests/integration/dogshell/test_dogshell.py::TestDogshell::test_screenboards - AssertionError: b''
datadog> FAILED tests/integration/dogshell/test_dogshell.py::TestDogshell::test_monitors - AssertionError: b''
datadog> FAILED tests/integration/dogshell/test_dogshell.py::TestDogshell::test_host_muting - AssertionError: b''
datadog> FAILED tests/integration/dogshell/test_dogshell.py::TestDogshell::test_downtime_schedule - AssertionError: b''
datadog> FAILED tests/integration/dogshell/test_dogshell.py::TestDogshell::test_downtime_cancel_by_scope - AssertionError: b''

--- a/pkgs/development/python-modules/datadog/default.nix
+++ b/pkgs/development/python-modules/datadog/default.nix
@@ -48,11 +48,12 @@ buildPythonPackage rec {
 
   disabledTestPaths = [
     "tests/performance"
+    # https://github.com/DataDog/datadogpy/issues/800
+    "tests/integration/api/test_*.py"
   ];
 
   disabledTests = [
     "test_default_settings_set"
-  ] ++ lib.optionals (pythonAtLeast "3.11") [
     # https://github.com/DataDog/datadogpy/issues/746
     "TestDogshell"
   ];
@@ -62,7 +63,6 @@ buildPythonPackage rec {
   ];
 
   meta = with lib; {
-    broken = true; # https://github.com/DataDog/datadogpy/issues/800
     description = "The Datadog Python library";
     homepage = "https://github.com/DataDog/datadogpy";
     changelog = "https://github.com/DataDog/datadogpy/blob/v${version}/CHANGELOG.md";

